app/controllers/api/v1/users_controller.rb

```diff
@@ -1,21 +1,19 @@
 class Api::V1::UsersController < ApplicationController
-
   def index
     sso_cookie = cookie_data
-    if sso_cookie.blank?
-      return return_bad_request('User')
-    end
-    #get opportunity
-    opportunity = Opportunity.where(os_accounts_id: sso_cookie.dig('sub', 'id'))
-    # get contact
-    contact = Contact.where(salesforce_id: sso_cookie.dig('sub', 'salesforce_contact_id'))
-    # get leads
-    lead = Lead.where(os_accounts_id: sso_cookie.dig('sub', 'id'))
+    return return_bad_request('User') if sso_cookie.blank?
+
+    opportunities = Opportunity.where(os_accounts_id: sso_cookie.dig('sub', 'id'))
+    contact = Contact.find_by(salesforce_id: sso_cookie.dig('sub', 'salesforce_contact_id'))
```

Comment:
We are, it was added here: https://github.com/openstax/accounts/pull/927/files#diff-6b2345c07385e696646a32e2669c3ff649c2f9d4dbc0aa91e26c38940511ab04R19

We do have [a card](https://app.zenhub.com/workspaces/openstax-bit-5bacfe3f4b5806bc2bea3764/issues/openstax/business-intel/1673) to change this to use the contact id from /api/user, because if the contact is changed and the user doesn't log out/in we won't see that.